lib/active_merchant/billing/gateways/usa_epay_transaction.rb

```diff
@@ -154,6 +155,13 @@ def add_address_for_type(type, post, credit_card, address)
         post[address_key(prefix, 'phone')]    = address[:phone]     unless address[:phone].blank?
       end
 
+      def split_names(full_name)
+        names = (full_name || '').split
```

Comment:
@aprofeit Yes, new guards have been added above to make it to use and return a consistent result e.g. `[nil, nil]`.
Thanks again.